docs/docs/plugins/autoformat.mdx

```diff
@@ -168,3 +39,74 @@ into an H1.
+| `autoformatArrow`              | Converts `->` to `→`.                                        |
+|                                | Converts `<-` to `←`.                                        |
+|                                | Converts `=>` to `⇒`.                                        |
+|                                | Converts `<=` and `≤=` to `⇐`.                               |
+| `autoformatLegal`              | Converts `(tm)` and `(TM)` to `™`.                           |
+|                                | Converts `(r)` and `(R)` to `®`.                             |
+|                                | Converts `(c)` and `(C)` to `©`.                             |
+| `autoformatLegalHtml`          | Converts `&trade;` to `™`.                                   |
+|                                | Converts `&reg;` to `®`.                                     |
+|                                | Converts `&copy;` to `©`.                                    |
+|                                | Converts `&sect;` to `§`.                                    |
+| `autoformatComparison`         | Converts `!>` to `!>`.                                       |
+|                                | Converts `!<` to `≮`.                                        |
+|                                | Converts `>=` to `≥`.                                        |
+|                                | Converts `<=` to `≤`.                                        |
+|                                | Converts `!>=` to `≱`.                                       |
+|                                | Converts `!<=` to `≰`.                                       |
+| `autoformatEquality`           | Converts `!=` to `≠`.                                        |
+|                                | Converts `==` to `≡`.                                        |
+|                                | Converts `!==` and `≠=` to `≢`.                              |
+|                                | Converts `~=` to `≈`.                                        |
+|                                | Converts `!~=` to `≉`.                                       |
+| `autoformatFraction`           | Converts `1/2` to `½`.                                       |
+|                                | Converts `1/3` to `⅓`.                                       |
+|                                | ...                                      |
+|                                | Converts `7/8` to `⅞`.                                       |
+| `autoformatDivision`           | Converts `//` to `÷`.                                        |
+| `autoformatOperation`          | Converts `+-` to `±`.                                        |
+|                                | Converts `%%` and `%%%` to `‰`.                              |
+|                                | `autoformatDivision` rules.                                  |
```

Comment:
@zbeyens I noticed you consolidated the 2 rules below into this one rule?
| Pattern | Result |
| :------: | :-----: |
| `%%` |	`‰` |
| `%%%` or `‰%` | `‱` |

Reply:
Typos, thanks for the review!